conf.py

```diff
@@ -104,6 +105,15 @@
 
 todo_include_todos = True
 
+# sphinxext-opengraph config
+ogp_site_url = "https://devguide.python.org/"
+ogp_site_name = "Python Developer's Guide"
+ogp_image = "https://devguide.python.org/_static/og-image.png"
```

Comment:
The immage is currently broken in the preview build.  This should fix it and should work once deployed too:
```suggestion
ogp_image = "_static/og-image.png"
```

Reply:
Updated.

With relative, it's still broken in preview builds because it's joined to `ogp_site_url = "https://devguide.python.org/"` and we don't yet have the image in `main`:

```html
<meta property="og:image" content="https://devguide.python.org/_static/og-image.png" />
```

But not a big problem, it'll be there soon :)

For testing, we can temporarily point `ogp_site_url` to the RTD preview URL. Feel free to push to this PR if you like.